mixer/pkg/protobuf/yaml/dynamic/encoder_test.go

```diff
@@ -702,17 +718,17 @@ func testMsg(t *testing.T, input string, output string, res protoyaml.Resolver,
 	if err != nil {
 		t.Fatalf("unable to decode: %v", err)
 	}
+	t.Logf("ff2 = %v", ff2)
 
 	// confirm that codegen'd code direct unmarshal and unmarhal thru bytes yields the same result.
 
 	if !reflect.DeepEqual(ff2, ff1) {
 		s, _ := diff.PrettyDiff(ff2, ff1)
 		t.Logf("difference: %s", s)
 		t.Fatalf("\n got: %v\nwant: %v", ff2, ff1)
+	} else {
+		t.Logf("\n got: %v\nwant: %v", ff2, ff1)
```

Comment:
why are we printing in case of success

Reply:
I left it there so I see success cases, but it should not be printed for success.
I will remove it later.